tilecloud_chain/__init__.py

```diff
@@ -666,11 +666,20 @@ def log_tiles(tile):
             self.imap(Logger(logger, logging.INFO, '%(tilecoord)s'))
 
     def add_metatile_splitter(self):
-        store = MultiTileStore({lname: MetaTileSplitterTileStore(
-            layer['mime_type'],
-            layer['grid_ref']['tile_size'],
-            layer['meta_buffer'])
-            for lname, layer in self.layers.items()})
+        class NullSplitter(TileStore):
+            @staticmethod
+            def get_one(tile):
+                return tile
+
+        splitters = {None: NullSplitter()}
+        for lname, layer in self.layers.items():
+            if layer.get('meta'):
```

Comment:
if the layer has no meta, it will have no entry in the splitters dict and default to the NullSplitter